src/components/product/OutOfStock/out-of-stock.scss

```diff
@@ -0,0 +1,39 @@
+@import "src/styles/scaffold";
+
+[data-store-out-of-stock] {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  margin-top: 24px;
+}
+
+[data-store-out-of-stock-input] {
+  width: 100%;
+  margin-top: var(--space-3);
```

Comment:
The new tokens scheme is incompatible with these variables. To get the new syntax, look at `globals/tokens.scss`.